.github/workflows/ci.yml

```diff
@@ -179,15 +175,14 @@ jobs:
       - name: create and activate env
         uses: mamba-org/provision-with-micromamba@main
         with:
-          environment-file: environment-dev.yml
```

Comment:
Why is the environment file not needed here?

Reply:
In general the use of the `environment-dev.yml` for the CI is not ideal, because it installs many dependencies that we don't need for testing...